crates/net/src/sim2h_worker.rs

```diff
@@ -94,6 +95,7 @@ impl Sim2hWorker {
             initial_authoring_list: None,
             initial_gossiping_list: None,
             has_self_stored_authored_aspects: false,
+            is_full_sync_DHT: true,
```

Comment:
needs to match the mode of the sim2h_server we've connected to... we could add a DhtAlgorithm to the same wire-message that lets us compare versions for sim2h compatibility.

Reply:
added to tree